components/support/support.jsx

```diff
@@ -4,25 +4,34 @@ import './support-style';
 
 export default class Support extends React.Component {
   render() {
-    let { number, type } = this.props;
+    let { minimum, maximum, size, type } = this.props;
     let supporters = require(`./support-${type}.json`);
 
     if (type === 'sponsors') {
+      supporters = supporters.slice();
       supporters.push(...Additional);
       supporters.sort((a, b) => b.totalDonations - a.totalDonations);
     }
 
+    if (typeof minimum === 'number') {
+      supporters = supporters.filter(item => item.totalDonations >= minimum * 100);
+    }
+
+    if (typeof maximum === 'number') {
+      supporters = supporters.filter(item => item.totalDonations < maximum * 100);
+    }
+
     return (
       <div className="support">
         {
-          supporters.slice(0, number).map((supporter, index) => (
```

Comment:
Just want to make sure this is intentional -- we don't want to limit the backers or supporters displayed at all?